targets/target_persistent.c

```diff
@@ -58,6 +127,8 @@ int main(int argc, char* argv[]) {
 
     gum_stalker_unfollow_me(stalker);
 
+    prefetch_write(compile_pipes[STDOUT_FILENO], prefetch_compiled);
```

Comment:
You could consider omitting the garbage collection step as you are about to bin the child anyway. You might get some performance gain.